lib/spack/spack/environment/environment.py

```diff
@@ -2957,33 +2956,28 @@ def __str__(self):
         return str(self.manifest_file)
 
     @property
-    def env_config_scopes(self) -> List[spack.config.ConfigScope]:
+    def env_config_scope(self) -> spack.config.ConfigScope:
         """A list of all configuration scopes for the environment manifest. On the first call this
         instantiates all the scopes, on subsequent calls it returns the cached list."""
```

Comment:
Almost forgot this .. (thanks for the reminder co-pilot!). The docstring needs to reflect the change here.